go-kit/metrics/provider.go

```diff
@@ -22,5 +22,6 @@ type Provider interface {
 	NewCounter(name string) metrics.Counter
 	NewGauge(name string) metrics.Gauge
 	NewHistogram(name string, buckets int) metrics.Histogram
+	NewUniqueCounter(name string) UniqueCounter
```

Comment:
Since the addition of this method makes it deviate from the documented Provider please update the docs above.